sim/priest/items.go

```diff
@@ -89,22 +89,22 @@ var ItemSetSanctificationRegalia = core.NewItemSet(core.ItemSet{
 		2: func(agent core.Agent) {
 		},
 		4: func(agent core.Agent) {
-			priest := agent.(PriestAgent).GetPriest()
```

Comment:
I think you can revert this change entirely. It seems likely that activating after the PWS cast is correct behavior.

Reply:
Agreed.